src/components/schemas/department.yaml

```diff
@@ -24,11 +24,19 @@ department_out:
       type: string
       nullable: true
       example: Outbound Sales
-    # doc_url:
-    #   type: string
-    #   nullable: true
-    #   description: |
-    #     Link to an external document that would contain documentation to be shared with employees
+    department_head_user_id:
```

Comment:
why are we not considering array of approvers like in projects

Reply:
This is fixed, pls check once @kartikeyrajvaidya